v4-client-js/examples/cancel_example.ts

```diff
@@ -0,0 +1,71 @@
+import { BECH32_PREFIX, OrderFlags } from '../src';
+import { CompositeClient } from '../src/clients/composite-client';
+import {
+  Network, OrderExecution, OrderSide, OrderTimeInForce, OrderType,
+} from '../src/clients/constants';
+import LocalWallet from '../src/clients/modules/local-wallet';
+import { Subaccount } from '../src/clients/subaccount';
+import { randomInt } from '../src/lib/utils';
+import { DYDX_TEST_MNEMONIC, MAX_CLIENT_ID } from './constants';
+
+async function sleep(ms: number): Promise<void> {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+async function test(): Promise<void> {
+  const wallet = await LocalWallet.fromMnemonic(DYDX_TEST_MNEMONIC, BECH32_PREFIX);
+  console.log(wallet);
+  const network = Network.staging();
+  const client = await CompositeClient.connect(network);
+  console.log('**Client**');
+  console.log(client);
+  const subaccount = new Subaccount(wallet, 0);
+
+  const longTermOrderClientId = randomInt(MAX_CLIENT_ID);
+  try {
+    // place a long term order
+    const tx = await client.placeOrder(
+      subaccount,
+      'ETH-USD',
+      OrderType.LIMIT,
+      OrderSide.SELL,
+      40000,
+      0.01,
+      longTermOrderClientId,
+      OrderTimeInForce.GTT,
+      60,
+      OrderExecution.DEFAULT,
+      false,
+      false,
+    );
+    console.log('**Long Term Order Tx**');
+    console.log(tx.hash);
+  } catch (error) {
+    console.log('**Long Term Order Failed**');
+    console.log(error.message);
+  }
+
+  await sleep(5000);  // wait for placeOrder to complete
```

Comment:
Can you note that for stateful orders, it can only be canceled after it's added to a block (which i'm assuming is why there's a sleep here)

Maybe also add a log statement "waiting for long-term order placement to included in a block"?